src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -49,6 +50,8 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
 
       if (isPositive && isDefined(exchangeRate)) {
         positiveAmountSum += parsedAmount.toNumber() * exchangeRate;
+      } else if (isDefined(exchangeRate)) {
```

Comment:
extract `isDefined(exchangeRate)` into upper case to make less check for the `false` case:
```
if (isDefined) {
    if (isPositive) {
        ...
    } else { .... }
}
```